packages/forms/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -115,6 +119,7 @@ const RadioGroup = ({
                   defaultChecked={defaultSelected === value}
                 />
               </InputWrapper>
+              {contentBelow && contentBelow}
```

Comment:
We will need this content associated with the input somehow for ATs. I'd reccomend assigning the content and `id` and then passing that to the radios `aria-describedby` attribute.

Reply:
This expression looks very weird to me.  Is there a reason for the nullish check?

Reply:
> This expression looks very weird to me. Is there a reason for the nullish check?

Haha might be an old school react/jsx thing. Check if the value is true, if so then display the content. Anyways, it's gone now after the changes Eric suggested were made. 





> We will need this content associated with the input somehow for ATs. I'd reccomend assigning the content and `id` and then passing that to the radios `aria-describedby` attribute.

@esizer I made the changes in [aaf8c64](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6541/commits/aaf8c64fd092080a7b7e2db3ab821c4168087ff5).